Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionAssetManager.cs

```diff
@@ -198,7 +198,7 @@ internal void SaveChangesToAsset()
             // If this is invoked after a domain reload, importAsset will resolve itself.
             // However, if the asset do not exist importedAsset will be null and we cannot complete the operation.
             if (importedAsset == null)
-                throw new Exception("Unable to save changes. Associated asset do not exist.");
+                throw new Exception("Unable to save changes. Associated asset does not exist.");
```

Comment:
Thanks for correcting this, but just wanted to highlight that InputActionAssetManager is currently only used for IMGUI editor, i.e. InputActionEditorWindow